tasks/main.yaml

```diff
@@ -231,14 +247,14 @@
     name: FreshClam
     special_time: "{{ clamav_freshclam_cron_time }}"
     job: "{{ clamav_freshclam_cron_job }}"
-  when: clamav_freshclam_cron == true
+    state: "{{ (clamav_freshclam_cron == true) | ternary('present', 'absent') }}"
```

Comment:
I would leave the `when: clamav_freshclam_cron == true` and set `preset` if `clamav_client_only` is `true`.

Reply:
Doing so prohibits Ansible from removing the cronjob if ClamAV changes from server to client-only.